app/assets/stylesheets/sass/abstracts/_variables.css.scss

```diff
@@ -0,0 +1,8 @@
+// Color variables
+$normal_Cyan: #35858B;
+$primaryColor: #AEFEFF;
+$very_Dark_Cyan: #072227;
+$very_pale_cyan: #4FBDBA;
+
+// Font family
+$font-stack: 'Quicksand', sans-serif;
```

Comment:
your variable names have different formats ($primaryColor, $normal_Cyan, $font-stack)
you should use only one format

Reply:
Change made